### ISSUE TITLE
AFC-1445 - Add context to the "Create", "Edit" - buttons in views

### ISSUE THREAD

**Issue by FredrikArvas** [vertelab/odoo-af#399]
**Userstory:** 
As a user, I want the standard-buttons texts to contain a context, because it helps me when I see the button of what is created

The text is added in the web-module and are global for all forms. 

**Screen-shots with changes:** (illustration of proposed changes)
![image](https://user-images.githubusercontent.com/9428370/99888016-6560a780-2c49-11eb-81b2-32a6b9360620.png)
![image](https://user-images.githubusercontent.com/9428370/99888046-be304000-2c49-11eb-8da4-5a6371aabe6a.png)

The idea here is to see if we could add a dynamic content through the translation. I have seen code in the translation like %s and wonder if we could create a %module which would enter a word that is specific to the module where the button is displayed.
![image](https://user-images.githubusercontent.com/9428370/99888114-6cd48080-2c4a-11eb-9114-1be2e04dbb9e.png)

**Module name:** (added by Nils or Fredrik)
odoo-web/web_a11y_filter_view

**Branch name:** 
Dev-12.0-CRM-AFC-1445-Web-A11y-Filter-views

**Module type:** 
Property module [see https://github.com/vertelab/odoo-af/issues/178 for more info]

**Estimate:** (added by developers only)
Estimated Hours: 
Estimated Date: 
Actual Hours:

**Comment by FredrikArvas** [comment]
Please comment on the possible solution here before we do development.

**Comment by rupareliyahemangi145** [comment]
Ok @FredrikArvas

**Comment by rupareliyahemangi145** [comment]
Hello @FredrikArvas 

I've checked this issue.Create and Edit string is comes fro XML template <t t-name="ListView.buttons">.

Here if we can get model description then we can achieve functionality which mentioned in issue. Because model description is transferable. But for now just for testing I've tried to change sting statically from Create to another and upgraded web module but it doesn't works.

I need to investigate more about this.

Spent time: 40 min today.

Thanks.

Regards,
Hemangi.

**Comment by FredrikArvas** [comment]
Thanks @rupareliyahemangi145 
I will ask some of our other community-developers if they have seen any of this. If not, then I will ask you to continue.
/Fredrik

**Comment by rupareliyahemangi145** [comment]
Ok @FredrikArvas

**Comment by FredrikArvas** [comment]
Hi again, @rupareliyahemangi145 
Nils spoke to the other developers and they did not know how to solve this. 
Please use up to two more hours in research. 
Thanks

**Comment by rupareliyahemangi145** [comment]
Ok @FredrikArvas Thanks. I will start research.

**Comment by rupareliyahemangi145** [comment]
Hello @FredrikArvas 

Created PR https://github.com/vertelab/odoo-web/pull/5 for this issue.

Here are screenshots:

**Contact views in English language:**

![kanban](https://user-images.githubusercontent.com/36560391/100076488-43dd0700-2e67-11eb-90d8-72eaf0014275.png)

![list_view](https://user-images.githubusercontent.com/36560391/100076507-493a5180-2e67-11eb-89d4-889f9d06c149.png)

![form_view](https://user-images.githubusercontent.com/36560391/100076519-4d666f00-2e67-11eb-892e-61c3c007bc6a.png)

**CRM views in English language:**

![crm_kanban](https://user-images.githubusercontent.com/36560391/100076577-5ce5b800-2e67-11eb-8cc6-ebc01a95b362.png)

![crm_list](https://user-images.githubusercontent.com/36560391/100076592-5fe0a880-2e67-11eb-98d1-6a72df614d33.png)

![crm_form](https://user-images.githubusercontent.com/36560391/100076609-640cc600-2e67-11eb-92e0-2814060794a0.png)

**CRM views in Swedish language:**

![crm_form_swedish](https://user-images.githubusercontent.com/36560391/100076651-6ff88800-2e67-11eb-8a57-8232556c85b3.png)

![crm_kanaban_swedish](https://user-images.githubusercontent.com/36560391/100076663-72f37880-2e67-11eb-99a5-65342f3b2b0f.png)

![crm_list_swedish](https://user-images.githubusercontent.com/36560391/100076675-78e95980-2e67-11eb-8387-75faacc87ba1.png)

Please check it and let me know if anything.

Thanks.

Spent time: 2 hours and 10 minutes

Regards,
Hemangi.

**Comment by FredrikArvas** [comment]
Thanks @rupareliyahemangi145 
This will be interesting to see how this works in the complete system.
@Viltersten Please test this.

**Comment by rupareliyahemangi145** [comment]
Ok @FredrikArvas
